src/net/sourceforge/kolmafia/session/QuestManager.java

```diff
@@ -1682,423 +1681,469 @@ public static void updateQuestData(String responseText, String monsterName) {
+        QuestDatabase.setQuestProgress(Quest.GOBLIN, QuestDatabase.FINISHED);
+      }
+      case "Jayden Belmont", "Very Dry Bones", "Robonerdagon" -> {
+        QuestDatabase.setQuestProgress(Quest.CYRPT, "step1");
+      }
+      case "Sharona", "Angry Sun", "Groarbot" -> {
+        QuestDatabase.setQuestProgress(Quest.TRAPPER, "step5");
+      }
+      case "smut orc jacker", "smut orc nailer", "smut orc pipelayer", "smut orc screwer" -> {
+        // Out of the corner of your eye, you see another smut orc shiver and
+        // disappear into a nasty-looking shack.
+        if (responseText.contains("another smut orc shiver")) {
+          Preferences.increment("smutOrcNoncombatProgress", 1, 15, false);
+        }
+        // Out of the corner of your eye, you see two smut orcs give one
+        // another a meaningful glance, shiver, and disappear into a nearby
+        // shack.
+        else if (responseText.contains("you see two smut orcs")) {
+          Preferences.increment("smutOrcNoncombatProgress", 2, 15, false);
+        }
+        // You see a nearby group of smut orcs huddling together for
+        // warmth. You are glad they are as far away as they are.
+        else if (responseText.contains("smut orcs huddling together")) {
+          Preferences.increment("smutOrcNoncombatProgress", 3, 15, false);
+        }
+        // You hear a bunch of windows being slammed shut against the cold.
+        else if (responseText.contains("windows being slammed shut")) {
+          Preferences.increment("smutOrcNoncombatProgress", 4, 15, false);
+        }
+        // Dozens of nearby smut orcs, shivering in the cold, rush into their
+        // shacks and slam the doors.
+        else if (responseText.contains("Dozens of nearby smut orcs")) {
+          Preferences.increment("smutOrcNoncombatProgress", 5, 15, false);
+        }
+      }
+      case "The Thing with No Name" -> {
+        ResultProcessor.processResult(ItemPool.get(ItemPool.FURIOUS_STONE, -1));
+        ResultProcessor.processResult(ItemPool.get(ItemPool.VANITY_STONE, -1));
+        ResultProcessor.processResult(ItemPool.get(ItemPool.LECHEROUS_STONE, -1));
+        ResultProcessor.processResult(ItemPool.get(ItemPool.JEALOUSY_STONE, -1));
+        ResultProcessor.processResult(ItemPool.get(ItemPool.AVARICE_STONE, -1));
+        ResultProcessor.processResult(ItemPool.get(ItemPool.GLUTTONOUS_STONE, -1));
+
+        QuestDatabase.setQuest(Quest.CLUMSINESS, QuestDatabase.UNSTARTED);
+        QuestDatabase.setQuest(Quest.GLACIER, QuestDatabase.UNSTARTED);
+        QuestDatabase.setQuest(Quest.MAELSTROM, QuestDatabase.UNSTARTED);
+
+        Preferences.setInteger("lastThingWithNoNameDefeated", KoLCharacter.getAscensions());
+      }
+      case "The Superconductor" -> {
+        Preferences.setBoolean("superconductorDefeated", true);
+      }
+      default -> {
+        if ((monsterName.startsWith("Jerry Bradford") && monsterName.contains("World Champion"))
```

Comment:
checking the Git history, I think the initial problem was that the name was given as `Jerry Bradford, Pokéfam World Champion`, so it could probably go inline as `Jerry Bradford, Pok&eacute;fam World Champion`

Reply:
That makes sense. Done.

Doing so, leaves only checking the Thwaitgold bee statue in the "default" section. That's because Bees Hate You final boss is The Guy Made of Bees. So, I made a case statement for that monster and got rid of the default. With a comment.